### ISSUE TITLE
Crop's HarvestMaxIncreasePerFarmingLevel and ExtraHarvestChance should be floats

### ISSUE THREAD

**Issue by Dyanosis** [scriptsforweirdos/StardewJA2ConPat#1]
The following 2 fields in Crop should be floats as they are percentages. You can see examples in Data/Crops on things like Potatoes and Blueberries. For HarvestMaxIncreasePerFarmingLevel, you can see an example with Rice.

```
HarvestMaxIncreasePerFarmingLevel: Optional[float] = 0
ExtraHarvestChance: Optional[float] = 0
```

**Comment by scriptsforweirdos** [comment]
Yes indeed they should be. Will tweak although next push will be very different from the current version.
